includes/admin/views/tmpl-order-actions.php

```diff
@@ -49,6 +55,8 @@ class="button button-secondary"
 			<span class="edd-help-tip dashicons dashicons-unlock" title="<?php esc_attr_e( 'The refund window for this Order has passed; however, you have the ability to override this.', 'easy-digital-downloads' ); ?>"></span>
 		<?php elseif ( false === $is_refundable && true === $is_window_passed ) : ?>
 			<span class="edd-help-tip dashicons dashicons-lock" title="<?php esc_attr_e( 'The refund window for this Order has passed.', 'easy-digital-downloads' ); ?>"></span>
+		<?php elseif ( 'amazon' === $order->gateway ) : ?>
+			<span class="edd-help-tip dashicons dashicons-lock" title="<?php esc_attr_e( 'Orders placed through the Amazon gateway must be refunded through Amazon.', 'easy-digital-downloads' ); ?>"></span>
```

Comment:
We should maybe add that after they have processed the payment as 'refunded' at the gateway, they need to come back here and manually set the payment status to 'Refunded'

Reply:
I thought about that--feels a little problematic because it's really all hidden in the tooltip. I'd prefer not to use a tooltip at all, honestly, at least not for this message.